src/components/BuildPages/BuildPages.tsx

```diff
@@ -127,14 +129,19 @@ export const BuildPages = () => {
 
   const isLogged = !serviceContainerBuild.data?.status || isBuildWithLog(serviceContainerBuild.data.status);
   const isLoggedTooltip = !isLogged ? `Builds with status ${serviceContainerBuild.data!.status} are not logged.` : '';
+  const liveLogLinkTooltip = !isBuilding ? `Not available while not building.` : '';
+  const staticLogLinkTooltip = isBuilding ? `Not available while building.` : '';
```

Comment:
I would go with the same message which is already displayed on details tab: `Build is not finished yet`

![image](https://github.com/user-attachments/assets/c30e3c33-4ec4-4eec-b48b-c3605fdcc239)